src/app/cart/page.tsx

```diff
@@ -1,14 +1,42 @@
 "use client"
 import Button from '@/components/Button'
 import { useCartStore } from '@/utils/store'
+import { useSession } from 'next-auth/react'
+import { useRouter } from 'next/navigation'
 import React, { useEffect } from 'react'
 
 const CartPage = () => {
 
+  const {data:session} = useSession()
+  const router = useRouter()
+
   useEffect(() => {
     useCartStore.persist.rehydrate()
   }, [])
 
+  const handleChckout = async () => {
+    if(!session){
+      router.push("./")
+    } else {
+      try {
+        const res = await fetch("http://localhost:3000/api/orders", {
+          method:"POST",
+          headers:{"Content-Type":"application/json"},
+          body: JSON.stringify({
+            price: totalPrice,
+            products,
+            status: "Not paid!",
```

Comment:
You can create an enum that will have every status. Can you explain me why we need to do so?